activiti-spring-process-extensions/src/test/java/org/activiti/spring/process/ProcessExtensionTestApplication.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.spring.process;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class ProcessExtensionTestApplication {
```

Comment:
why do we need a spring boot application test in this common module?

Reply:
The test is not in the common module, it's in the extensions file module. The tests were already in place before, but they were referencing specific configurations to load. Now that the data format provider was moved in another auto-configuration, we would need to add it to the list of configurations to load or let spring boot load the ones available. I preferred the second approach as it's easier to maintain.